src/mappers/flags/flag_mapper_builder.h

```diff
@@ -0,0 +1,29 @@
+#ifndef SRC_MAPPERS_FLAGS_FLAGMAPPERBUILDER_H
+#define SRC_MAPPERS_FLAGS_FLAGMAPPERBUILDER_H
+
+#include <filesystem>
+#include <set>
+#include <string>
+
+#include "external/commonItems/ModLoader/ModFilesystem.h"
+#include "src/mappers/flags/flag_mapper.h"
+
+
+namespace mappers
+{
+class FlagMapperBuilder
+{
+  public:
+   void ReadConfig(const std::string& config_file);
+   bool CreateTargetFolders(const std::string output_name);
```

Comment:
Can this be a static function or a non-member function?

Reply:
Sorry, not quite getting this? Both methods read information that's specific to the builder object and will be passed into the mapper it builds; `forbid` and `custom_flags` in one case, `base_folder` in the other. Can you expand?

Reply:
Maybe I misread it. On my earlier pass it looked like CreateTargetFolders() only used local variables and parameters, not member variables. If that's untrue, disregard.

Reply:
Yes, it also sets the `base_folder_` member variable. I'll admit that there's probably a more elegant way to do that.